pkgs/top-level/all-packages.nix

```diff
@@ -10257,10 +10257,12 @@ in
     stripped = false;
   }));
 
-  crossLibcStdenv = overrideCC stdenv
-    (if stdenv.hostPlatform.useLLVM or false
-     then buildPackages.llvmPackages_8.lldClangNoLibc
-     else buildPackages.gccCrossStageStatic);
+  gccCrossLibcStdenv = overrideCC stdenv buildPackages.gccCrossStageStatic;
+
+  crossLibcStdenv =
+    if stdenv.hostPlatform.useLLVM or false
+    then overrideCC stdenv buildPackages.llvmPackages_8.lldClangNoLibc
```

Comment:
BTW this should use the default LLVM version now that it's updated to be newer for cross.

Reply:
It's still clang 7 for `useLLVM` right? I think we'll have to check up on this topic anyways. Unfortunately there's still *some* issue with `llvmPackages_7.lldClang` with C++ and libunwind (i. e. `ncurses`' C++ library doesn't compile with `useLLVM` atm).

This is probably something unrelated to this though, I'd expect we can compile musl with clang 7 just fine. I'll try to rememeber to check that later.

Reply:
I think it's a later version when cross compilation is involved. We can tweak the condition if something isn't working.

Reply:
At least `stdenv` itself still uses clang 7.